src/librustc/ty/layout.rs

```diff
@@ -151,9 +189,9 @@ pub enum Endian {
 }
 
 /// Size of a type in bytes.
-#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
 pub struct Size {
-    raw: u64
+    pub bytes: u64
```

Comment:
I don't really care, but I imagine the use of privacy here was to help ensure people don't do stupid things like `let mut size = ...; size.bytes *= 2;`, but rather encourage them to go through these (presumably more careful) APIs.